api/python/provisioner/commands/destroy.py

```diff
@@ -304,6 +308,11 @@ def run(self, **kwargs):  # noqa: C901
                 logger.info("Teardown the system states")
                 self._run_states('system', run_args)
 
+            if 'utils' in run_args.states:
+                logger.info("Teardown foundation states")
+                self._run_states('utils', run_args)
+                self._run_cmd(list_cmds)
```

Comment:
I guess that should be run only once at the very end but the current code makes it possible to be called few times

Reply:
thanks for pointing, removed!